packages/lib/src/lua/net/websocket.rs

```diff
@@ -144,19 +152,24 @@ async fn close<'lua, T>(
 where
     T: AsyncRead + AsyncWrite + Unpin,
 {
-    let mut ws = socket.stream.lock().await;
-    let res = ws.close(Some(WsCloseFrame {
-        code: match code {
-            Some(code) if (1000..=4999).contains(&code) => WsCloseCode::from(code),
-            Some(code) => {
-                return Err(LuaError::RuntimeError(format!(
-                    "Close code must be between 1000 and 4999, got {code}"
-                )))
-            }
-            None => WsCloseCode::Normal,
-        },
-        reason: "".into(),
-    }));
+    let mut ws = socket.stream_out.lock().await;
+
+    let _ = ws
```

Comment:
I don't think we should ignore the result here, in case sending the close message fails. You can map the error like we do for the return value, and use the `?` operator to bubble it up

Reply:
Issue has been addressed in the [e5a844e](https://github.com/filiptibell/lune/pull/68/commits/e5a844ecec22c10cc4ee472f531bde6511abe07a) commit